pkg/controller/sriovnetwork/sriovnetwork_controller.go

```diff
@@ -93,6 +93,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 }
 
 var _ reconcile.Reconciler = &ReconcileSriovNetwork{}
+var lastNetworkNamespace = ""
```

Comment:
Will `lastNetworkNamespace` var be overwritten if there are multiple SriovNetwork?

Reply:
Fixed.